### PR TITLE
FT-Guides to 6.3.0-build

### PR DESCRIPTION
**Pull request by MichaelShn**
FT Guide

TF getting started guide

Update tf_geting_started.md

New files

Update images

Update tf_geting_started.md

Images Path

update

Update tf_geting_started.md

Update tf_geting_started.md

Update tf_geting_started.md

Update tf_geting_started.md

Update tf_geting_started.md

Titles

Update tf_geting_started.md

Titles

Update tf_marketplace.md

### CODE REVIEW

**Review by ilanyaniv — APPROVED** [review]

File: content/trial_getting_started/ft_start/tf_geting_started.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
++++
+
+title = "Create Blueprint from Terraform module"
+description = "Terraform Based blueprints- Creating a blueprint from a Terraform module deploying and Installing"
+weight = 27
+alwaysopen = false
++++
+
+Getting Started guide to Creating a blueprint from a terraform module deploying and Installing it.
```
Comment (ahmadiesa-abu) [inline content/trial_getting_started/ft_start/tf_geting_started.md:9]:
```suggestion
Getting Started guide to create a blueprint from a terraform module deploying and Installing it.
```

File: content/trial_getting_started/ft_start/tf_geting_started.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+## Prerequisites
+* Cloudify Manager Version 6.3 or later
+* Terraform module archive available from a URL
+* Credentials to Cloud provider
+
+
+## Overview
+In this guide we are going through following steps to create a blueprint from a terraform module and test it.
```
Comment (ahmadiesa-abu) [inline content/trial_getting_started/ft_start/tf_geting_started.md:18]:
```suggestion
In this guide we are going through the following steps to create a blueprint from a terraform module and test it.
```

File: content/trial_getting_started/ft_start/tf_geting_started.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+* Prepare the manager by defining cloud Credentials Secrets
+* Provide a URL to the Terraform archive and chose the right module
```
Comment (ahmadiesa-abu) [inline content/trial_getting_started/ft_start/tf_geting_started.md:21]:
```suggestion
* Provide a URL to the Terraform archive and choose the right module
```

File: content/trial_getting_started/ft_start/tf_geting_started.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+## Step 4 Define the variables for the Terraform module
+* Open the variables section and click the ‘Add’ button
+* Type the variable name
+* Select the source type and a value / input name / secret name
+  * Static value - A Value  that will always be assigned to the variable
+  * Input  - the variable that will be set by an input that will be presented when a deployment is created.
+  * Secret - The value assigned to the variable that will be taken from the secret store.
+
+If the Terraform module dose not have credentials set as variables, see below title "Using modules from Registry, Passing provider credentials by defining the Environment Variables"
```
Comment (ahmadiesa-abu) [inline content/trial_getting_started/ft_start/tf_geting_started.md:66]:
```suggestion
If the Terraform module does not have credentials set as variables, see below title "Using modules from Registry, Passing provider credentials by defining the Environment Variables"
```

File: content/trial_getting_started/ft_start/tf_geting_started.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+
+## Step 6 Create the blueprint
+Submit the dialog, errors discovered will appear in the dialog. After successful submission you will be forwarded to the blueprint page.
+
+## Step 7 Create a Deployment and Test the blueprint
+* Click the ‘create deployment’ button.
+* Review the logs, located in the logs section of the deployment page
+* Review deployment output
+* Review the inputs in the Deployment Creation dialog
+* Inspect the install workflow execution graph until it will finished, The bars of the execution graph will turn green when operations will finish.
```
Comment (ahmadiesa-abu) [inline content/trial_getting_started/ft_start/tf_geting_started.md:91]:
```suggestion
* Inspect the install workflow execution graph until it finishes, The bars of the execution graph will turn green when operations will finish.
```

File: content/trial_getting_started/ft_start/tf_geting_started.md [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+AWS Example:
+
+* Environment Variable: AWS_ACCESS_KEY_ID  Type: Secret  Secret's Name: aws_access_key_id
+* Environment Variable: AWS_SECRET_ACCESS_KEY Type: Secret  Secret's Name: aws_secret_access_key
+* Environment Variable: AWS_DEFAULT_REGION Type: static Value: us-west-2
+
+## Troubleshooting
+
+* The credentials were defined in the variables section instead of the environment variables section
+ * Error: Value for undeclared variableThe root module does not declare a variable named "AWS_ACCESS_KEY_ID" but a value was found in…
```
Comment (ahmadiesa-abu) [inline content/trial_getting_started/ft_start/tf_geting_started.md:109]:
```suggestion
 * Error: Value for undeclared variable The root module does not declare a variable named "AWS_ACCESS_KEY_ID" but a value was found in…
```

File: content/trial_getting_started/ft_start/tf_geting_started.md [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+* The Version of terraform selected was too old for the terraform module you have uploaded.
+ * Error: Could not load pluginPlugin reinitialization required. Please run "terraform init".
```
Comment (ahmadiesa-abu) [inline content/trial_getting_started/ft_start/tf_geting_started.md:111]:
```suggestion
 * Error: Could not load plugin Plugin reinitialization required. Please run "terraform init".
```

File: content/trial_getting_started/ft_start/tf_marketplace.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
++++
+
+title = "Get Terraform Blueprints from the Marketplace"
+description = "Get Terraform based blueprints from the marketplace"
+weight = 28
+alwaysopen = false
++++
+
+Getting Started guide to uploading and deploying Terraform based blueprints from the marketplace.
```
Comment (ahmadiesa-abu) [inline content/trial_getting_started/ft_start/tf_marketplace.md:9]:
```suggestion
Getting Started guide for uploading and deploying Terraform based blueprints from the marketplace.
```

File: content/trial_getting_started/ft_start/tf_marketplace.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+
+## Prerequisites
+* Cloudify Manager Version 6.3 or later
+* Cloud Provider Credentials (AWS in this case)
+
+
+## Overview
+In this guide we are going through following steps to upload a blueprint based on terraform from the marketplace and test it.
```
Comment (ahmadiesa-abu) [inline content/trial_getting_started/ft_start/tf_marketplace.md:17]:
```suggestion
In this guide we are going through the following steps to upload a blueprint based on terraform from the marketplace and test it.
```

File: content/trial_getting_started/ft_start/tf_marketplace.md [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+## Step 2 Test the blueprint by creating a Deployment and Installing it
+* Click The Create Deployment Button
+* Review the inputs in the Deployment Creation dialog
+* You would be prompted to define The Credentials to the cloud as secrets, those will be propagated to the Terraform Module
+
+![Terraform Marketplace]( /images/trial_getting_started/tf/TtMarketplace.jpg )
+
+
+## Step 3 Verify the installation.
+* Inspect the install workflow execution graph until it will finished, The bars of the execution graph will turn green when operations will finish.
```
Comment (ahmadiesa-abu) [inline content/trial_getting_started/ft_start/tf_marketplace.md:42]:
```suggestion
* Inspect the install workflow execution graph until it finishes, The bars of the execution graph will turn green when operations will finish.
```

**Review by ahmadiesa-abu — COMMENTED** [review]
left small comments

**Pull request merged**